### PR TITLE
e2e BigQuery Additional Test Scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
@rmstar @itsankit-google Please review.
cc @sawantpritam 

This PR is dependent on https://github.com/cdapio/cdap-e2e-tests/pull/75

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java [hunk 1]
```diff
@@ -172,6 +181,91 @@ public static void createTempPartitionedSourceBQTable() throws IOException, Inte
     BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
   }
 
+  /**
+   * Create BigQuery table with different list of dataypes (transaction_info BOOL, transaction_num BYTES,
+   * transaction_uid STRING, transaction_date DATE, transaction_dt DATETIME, transaction_time TIME, latitude FLOAT64,
+   * unique_key INT64, business_ratio NUMERIC, updated_on TIMESTAMP, parent ARRAY<STRING>,
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java:206]: Looks like bignumeric is missing. Please make sure we have coverage for all supported data types in source and sink.
https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryTable-batchsource.md#data-type-mappings-from-bigquery-to-cdap
https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryTable-batchsink.md#data-type-mappings-from-cdap-to-bigquery

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java [hunk 2]
```diff
@@ -172,6 +181,91 @@ public static void createTempPartitionedSourceBQTable() throws IOException, Inte
...
+   * inputs STRUCT<input_script_bytes BYTES, input_script_string STRING, input_sequence_number INT64>)
+   * containing random testdata.
+   */
+  @Before(order = 1, value = "@BQ_SOURCE_DATATYPE_TEST")
+  public static void createSourceBQTableWithDifferentDataTypes() throws IOException, InterruptedException {
+    createSourceBQTableWithQueries(PluginPropertyUtils.pluginProp("bqCreateTableQueryFile"),
+            PluginPropertyUtils.pluginProp("bqInsertDataQueryFile"));
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: Indent looks incorrect. Please reformat all files in the PR using this settings file - https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java (hunk 1 above)
Comment (itsmekumari) [reply src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java:206]: Changes done.

File: src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java (hunk 2 above)
Comment (itsmekumari) [reply src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java]: Changes done.

**Review by rmstar — APPROVED** [review]

**Pull request merged**